src/System.Net.Requests/tests/HttpWebRequestTest.cs

```diff
@@ -188,6 +188,31 @@ public void ContentType_SetThenGetEmptyValue_ExpectNull(Uri remoteServer)
             });
         }
 
+        [Fact]
+        public async Task HttpWebRequest_SetHostHeader_ContainsPortNumber()
+        {
+            await LoopbackServer.CreateServerAsync(async (server, uri) =>
+            {
+                HttpWebRequest request = WebRequest.CreateHttp(uri);
+                string host = uri.Host + ":" + uri.Port;
+                request.Host = host;
+                Task<WebResponse> getResponse = request.GetResponseAsync();
+
+                await server.AcceptConnectionAsync(async connection =>
+                {
+                    await connection.Reader.ReadLineAsync(); // Skip status line.
+                    Assert.Contains(host, await connection.Reader.ReadLineAsync());
+
+                    await connection.SendResponseAsync();
+                });
+
+                using (HttpWebResponse response = (HttpWebResponse)await getResponse)
```

Comment:
nit: since you are already casting the 'await' to an HttpWebResponse, you could then use 'var' when defining the local variable 'response'.

Reply:
I copied the pattern from other places in the same file.

I understand that we can use `var` in this case, does it has advantage over the concrete type in this case? Seems the compiler generates the exact same for both variants. Or it's just a preferred style?

Reply:
>I copied the pattern from other places in the same file.

In this case, the pattern in the other places is not-optimal.

No big deal. But 'var' is preferred to be used when the type is obvious.  And in cases where a 'new' or a cast to that type is used, 'var' should be used.  It is a preferred style.

cc: @stephentoub